### PR TITLE
Add support for overlapping sat regions 

### LINKED ISSUE #46: Add support for overlapping satellite CSV regions
Satellite CSV data can now be joined with iNaturalist observations (see #44). The satellite data may contain overlapping regions. This can result in multiple regions being matched for a single iNaturalist observation. When this occurs choose the region who's center is the closest to the observation lat/long.

### LINKED ISSUE #47: Show all observation columns
Additional columns can now be added to the iNaturalist observation data(see #44). Currently only the hard coded iNaturalist columns are visible. Change the UI to show all columns.

### PR DESCRIPTION
**Pull request by johnbradley**
When joining RGB or Landcoverage satellite data with iNaturalist observations and multiple regions are found return the region that has the closest centroid. Also adds code to display all observation columns adding horizontal scrolling support and making the label column sticky.

Fixes #46
Fixes #47

### CODE REVIEW

File: andromeda/satellitedata.py [hunk 1]
```diff
@@ -28,7 +28,18 @@ def read_satellite_data(url):
...
+    # if the point exists in multiple regions find the closest to the centroid
+    if len(matches.index) > 1:
+        # calculate the distance between the point and the regions
+        distances = sat_geo_series.centroid.distance(point)
+        # find the minimum distance
+        min_distance = min(distances)
+        # find the closest points
+        closest = distances == min_distance
+        # those are our matches
+        matches = sat_df[closest]
```
Comment (thompsonmj) [inline andromeda/satellitedata.py:41]: It's theoretically possible that `len(matches.index) > 1` still, but pretty unlikely.

**Review by thompsonmj — APPROVED** [review]

File: andromeda/satellitedata.py (hunk 1 above)
Comment (hlapp) [reply andromeda/satellitedata.py:41]: That would require that a location be exactly in the middle between two centroids, right?

Comment (thompsonmj) [reply andromeda/satellitedata.py:41]: Right, I think it's probably a negligible chance.

Comment (johnbradley) [reply andromeda/satellitedata.py:41]: There is a warning returned when there are multiple matches higher up in the code, but I doubt anyone will ever see the warning.

**Comment by johnbradley** [comment]
I'm going to close this PR and update the code based on https://github.com/Imageomics/Andromeda/issues/46#issuecomment-1632733185 and https://github.com/Imageomics/Andromeda/issues/46#issuecomment-1632735619

File: andromeda/tests/test_satellitedata.py [hunk 2]
```diff
@@ -12,10 +12,10 @@
 def sample_rgb_dataframe():
     columns = [LAT_NW, LON_NW, LAT_SE, LON_SE, "Number"]
     data = [
-        (40.332624,	-74.749758,	40.313732,	-74.707701, 111),
+        (40.332624,	-74.749758,	40.313732,	-74.707701, 111,),
```
Comment (egrace479) [inline andromeda/tests/test_satellitedata.py:15]:
```suggestion
        (40.332624,	-74.749758,	40.313732,	-74.707701, 111),
```

Comment (johnbradley) [reply andromeda/tests/test_satellitedata.py:15]: Thanks for finding this. I'll incorporate this into the next PR.

**Pull request closed without merging**